skaffold.yaml

```
@@ -591,7 +591,7 @@ build:
           gitCommit:
             variant: AbbrevCommitSha
   artifacts:
-    - image: europe-docker.pkg.dev/gardener-project/releases/gardener/extensions/provider-local
+    - image: gardener-extension-provider-local
```

In https://github.com/gardener/gardener/pull/9782, we just decided to rename them following this pattern: `local-skaffold/<name>`

```suggestion
    - image: local-skaffold/gardener-extension-provider-local
```